qemu/tests/cfg/block_iscsi_4kdisk.cfg

```diff
@@ -0,0 +1,50 @@
+- block_iscsi_4kdisk:
+    virt_test_type = qemu
+    type = block_iscsi_4kdisk
+    only Windows
+    start_vm = no
+    kill_vm = yes
+    kill_vm_gracefully = yes
+    kill_vm_on_error = yes
```

Comment:
we can remove kill_vm_on_error for it will not be used

Reply:
ok,will update